testcase/cross-region/tso.go

```diff
@@ -352,31 +405,33 @@ func (c *crossRegionClient) waitAllocatorReady(dcLocations []string) error {
 }
 
 func (c *crossRegionClient) waitLeader(name string) error {
-	return util2.WaitUntil(fmt.Sprintf("%s leader election", name), func() bool {
+	for {
```

Comment:
There will be no timeout

Reply:
I think it's unnecessary since the CI test of TiPocket will kill it automatically when it's timeout.